dds_web/api/user.py

```diff
@@ -533,11 +519,23 @@ class Token(flask_restful.Resource):
     @basic_auth.login_required
     @logging_bind_request
     def get(self):
-        return flask.jsonify({"token": jwt_token(username=auth.current_user().username)})
+        args = flask.request.json
+        if args is None:
+            args = {}
 
+        data = token_schemas.TokenSchema().load(args)
+
+        return flask.jsonify(
+            {
```

Comment:
I would keep the encrypted token as it is and only add 
`"message": "Please take this token to /user/second_factor to authenticate with MFA!",`

Reply:
For clarity, I meant that changes to encrypted token can be reverted and only the suggested line in the previous comment can be added.